Jenkinsfile

```diff
@@ -161,6 +162,15 @@ def uploadToJfrogReleases(pkg, fileName) {
     }
 }
 
+def copyToReleasesLatestDir() {
+    print "Copy version $version to releases latest dir."
+    withCredentials([string(credentialsId: 'jfrog-cli-automation', variable: 'JFROG_CLI_AUTOMATION_ACCESS_TOKEN')]) {
+        sh """#!/bin/bash
+                builder/jfrog rt cp jfrog-cli/$version/(*) jfrog-cli/latest/{1}" --flat --url https://releases.jfrog.io/artifactory/ --access-token=$JFROG_CLI_AUTOMATION_ACCESS_TOKEN
```

Comment:
```
...jfrog-cli/latest/{1}... --> ...jfrog-cli/latest_v1/{1}...
```
This will make it easier for us to install the latest version of JFrog CLI v1 (once we go out with v2).